README.md

```diff
@@ -438,26 +437,52 @@ This Query returns an [AnnahmeJob](#annahmejob). It contains the Status of the a
 
 ## Request Datatypes
 
-### Partner-ID
+### AngebotOptions
+
+    {
+        includeMachbarkeitsstatus: [MachbarkeitStatus]
+        includeVollstaendigkeitsstatus: [VollstaendigkeitStatus],
+        vertriebskanal: Vertriebskanal
+    }
+
+For possible value for `MachbarkeitStatus` see [hier](#machbarkeitstatus).
+
+For possible value for `VollstaendigkeitStatus` see [hier](#vollstaendigkeitstatus).
+
+#### Vertriebskanal
+
+    {
+        B2B2C
+        B2B
+    }
 
-This type is a 5-digit String and identifies a Plakette of the Europace-Partnermanagement.  
-The Partner-ID has to be underneath or identical to the Partner-ID of the API-Client.
 
 ### Datenkontext
 
 This type is a String which can currently have one of the following values
 
-* TESTUMGEBUNG
-* ECHTGESCHAEFT
+* `TESTUMGEBUNG`
+* `ECHTGESCHAEFT`
 
 ### Finanzierungszweck
 
 This type is a String which can currently have one of the following values
 
-* UMSCHULDUNG
-* FREIE_VERWENDUNG
-* FAHRZEUGKAUF
-* MODERNISIEREN
+* `UMSCHULDUNG`
```

Comment:
alle Enums in diesem Stil?

Reply:
Stimmt ich schreib das auch noch mit Klammern, wie bei den anderen.